src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -250,6 +252,9 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
       val recipeButton = createRecipeButton(recipe.title)
       recipeButton.addActionListener {
         GraphQlLogger.logCodyEvent(project, "recipe:" + recipe.id, "clicked")
+        val editorManager = FileEditorManager.getInstance(project)
```

Comment:
I simply added this call here - close to the action

Reply:
Could you explain why this is needed for this particular bug? Just as someone (me) who doesnt know the why on it, and for looking back on this issue + fix :slightly_smiling_face:

Reply:
The problem is looks like this:

1. IDE restarts
2. Agent gets up (initalising)
3. a few listeners (e.g. `editorCreated`) are called. however,
4. agnet is not yet connected!
5. `informAgentAboutEditorChange` ends at the connections check, `TextDocument` is not updated

See [isConnected](https://sourcegraph.com/github.com/sourcegraph/jetbrains/-/blob/src/main/kotlin/com/sourcegraph/cody/agent/CodyAgent.kt?L187)
See [informAgentAboutEditorChange](https://sourcegraph.com/github.com/sourcegraph/jetbrains/-/blob/src/main/kotlin/com/sourcegraph/cody/autocomplete/CodyEditorFactoryListener.kt?L147%3A9-147%3A37) and it's invocations 

is it clear to you? I can give more details if you need